src/cookbooks/parquet.md

```diff
@@ -1,65 +0,0 @@
-# Working with Parquet
-
-This guide will give you a quick introduction to working with
-[Parquet](https://parquet.apache.org/) files at Mozilla.
-You can also refer to Spark's documentation on the subject
-[here](http://spark.apache.org/docs/latest/sql-programming-guide.html#parquet-files).
-
-Most of our [derived datasets](../datasets/derived.md),
-like the `longitudinal` or `main_summary` tables,
-are stored in Parquet files.
-You can access these datasets in [re:dash](https://sql.telemetry.mozilla.org/),
-but you may want to access the data from an
-[ATMO](https://analysis.telemetry.mozilla.org/) cluster
-if SQL isn't powerful enough for your analysis
-or if a sample of the data will not suffice.
-
-## Table of Contents
-<!-- toc -->
-
-# Reading Parquet Tables
-
-Spark provides native support for reading parquet files.
-The result of loading a parquet file is a
-[DataFrame](http://spark.apache.org/docs/2.1.0/api/python/pyspark.sql.html#pyspark.sql.DataFrame).
-For example, you can load `main_summary` with the following snippet:
-
-```python
-# Parquet files are self-describing so the schema is preserved.
-main_summary = spark.read.parquet('s3://telemetry-parquet/main_summary/v1/')
-```
-
-You can find the S3 path for common datasets in
-[Choosing a Dataset](../concepts/choosing_a_dataset.md)
-or in the reference documentation.
-
-# Writing Parquet Tables
-
-Saving a table to parquet is a great way to share an intermediate dataset.
-
-## Where to save data
-
-You can save data to a subdirectory of the following bucket:
-`s3://net-mozaws-prod-us-west-2-pipeline-analysis/<username>/`
```

Comment:
This one line is still of some use while we're on AWS. Can you find somewhere else to put this info temporarily? Otherwise, 💯 !

Reply:
`src/tools/spark.md` might be a good place to add this for now.